docs/source/testing/ladybug/extendTable/extendTable.rst

```diff
@@ -46,4 +46,8 @@ To understand how this hack works, you have to know that the Frank!Framework is
 
 Ladybug is implemented by Java objects, some of them being Java beans that are instantiated by the Spring framework. Many of these beans are by default configured through file ``springIbisTestTool.xml``, which is part of the Frank!Framework. When you want to change Ladybug, you want to provide your own version of this file. This is achieved by setting the property ``ibistesttool.custom=Custom``, as you do in ``DeploymentSpecifics.properties``. This line tells the Frank!Framework to read ``springIbisTestToolCustom.xml`` instead of ``springIbisTestTool.xml``. You added ``springIbisTestToolCustom.xml`` to the ``classes`` folder.
 
-You do not want to change Ladybug completely. Many Java beans still have to be created like is defined in ``springIbisTestTool.xml``. This is achieved by the line ``<import resource="springIbisTestTool.xml"/>``. To add a column, you have to change the definitions of the following beans: ``metadataExtractor`` and ``metadataNames``. In ``springIbisTestToolCustom.xml``, you see the new definitions of these beans. These definitions specify what other objects have to be linked to the "metadataExtractor" and the "metadataNames" beans. Your new column is defined in two additional Java beans that are linked to the two mentioned beans. You see this in the highlighted lines of ``springIbisTestToolCustom.xml``.
\ No newline at end of file
+You do not want to change Ladybug completely. Many Java beans still have to be created like is defined in ``springIbisTestTool.xml``. This is achieved by the line ``<import resource="springIbisTestTool.xml"/>``. To add a column, you have to change the definitions of the following beans: ``metadataExtractor`` and ``metadataNames``. In ``springIbisTestToolCustom.xml``, you see the new definitions of these beans. These definitions specify what other objects have to be linked to the "metadataExtractor" and the "metadataNames" beans. Your new column is defined in two additional Java beans that are linked to the two mentioned beans. You see this in the highlighted lines of ``springIbisTestToolCustom.xml``.
+
+.. WARNING::
+
+   In case Ladybug needs data from the database, set ``jdbc.migrator.active=true`` in a properties file in the ``classes`` directory or in ``src/main/resources`` for a Maven project. Ladybug cannot read data that is specific to a configuration. See also subsection :ref:`testingLadybugStorages`.
```

Comment:
Last sentence can be removed
DatabaseChangelog.xml doesn't work without Configuration.xml, for jdbc.migrator.active this is not the case